src/main/kotlin/app/revanced/patches/youtube/layout/theme/patch/LithoThemePatch.kt

```diff
@@ -33,6 +35,19 @@ class LithoThemePatch : BytecodePatch(
                 move-result p1
             """
         )
+
+        fun buildInvokeString(player: Int) = """
+            const/4 v0, 0x$player
+            sput-boolean v0, Lapp/revanced/integrations/patches/LithoThemePatch;->isStandardPlayer:Z
+        """
+
+        StartVideoInformerFingerprint.result!!.mutableMethod.addInstructions(
```

Comment:
This will fail if the fingerprints are not resolved. Create a separate dependency patch which handles resolving them and the conjunction of their functionalities.

Reply:
Do it youself, thanks.

I have no intention of repeating previous experiences.

You know better than me what is needed to succesfully finish and merge the PR, otherwise close it and leave the bug issue opened...honestly I don't care.

Reply:
If there is no time for the PR the commit which caused the bug should be reverted, please open an issue for that instead.